python/cuml/cuml/accel/core.py

```diff
@@ -86,10 +86,24 @@ def install(disable_uvm=False):
         if _is_concurrent_managed_access_supported():
             import rmm
 
-            logger.debug("cuML: Enabling managed memory...")
-            rmm.mr.set_current_device_resource(rmm.mr.ManagedMemoryResource())
+            mr = rmm.mr.get_current_device_resource()
+            if isinstance(mr, rmm.mr.ManagedMemoryResource):
+                # Nothing to do
+                pass
+            elif not isinstance(mr, rmm.mr.CudaMemoryResource):
+                logger.debug(
+                    "cuML: A non-default memory resource is already configured, "
+                    "skipping enabling managed memory."
+                )
+            else:
+                rmm.mr.set_current_device_resource(
+                    rmm.mr.ManagedMemoryResource()
+                )
+                logger.debug("cuML: Enabled managed memory.")
         else:
-            logger.warn("cuML: Could not enable managed memory.")
+            logger.debug(
```

Comment:
I moved this log to debug - on platforms where managed memory isn't supported, there's no action a user can take to enable this. Having this viewable by default feels too noisy.

For comparison, by default `cudf.pandas` doesn't report anything on these platforms by default (they just don't setup managed memory).